### PR TITLE
Updating AS3 Schema to v3.30

### PR DESCRIPTION
**Pull request by Vidhi-Pat**
**Description**:  Updating AS3 Schema to v3.30

**Changes Proposed in PR**: Updating AS3 Schema to v3.30

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation

## CRD Checklist
- [x] Updated required CR schema

### CODE REVIEW

**Review by agupta49 — APPROVED** [review]
LGTM

File: schemas/as3-schema-3.30.0-5-cis.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json",
+  "$id": "urn:uuid:85626792-9ee7-46bb-8fc8-4ba708cfdc1d",
```
[the remark this answers is not in the document]
Comment (Vidhi-Pat) [reply schemas/as3-schema-3.30.0-5-cis.json]: Change reverted.

**Pull request merged**